etc/cherrytree.profile

```diff
@@ -0,0 +1,14 @@
+whitelist ${HOME}/cherrytree
+whitelist ${HOME}/.config/cherrytree/
+whitelist ${HOME}/.local/share/
+include /etc/firejail/disable-mgmt.inc
+include /etc/firejail/disable-secret.inc
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+caps.drop all
+seccomp
+protocol unix,inet,inet6,netlink
```

Comment:
Under help menu they have an "Online Manual". It directs the browser to http://giuspen.com/cherrytreemanual/Introduction.html